Add TestGrid Documentation and Linting (#10)
CSPI-QE/ocp-ci-docs: docs/OCP_CI_Tutorials/Scenarios/Execution_Guide.md

# OpenShift CI Scenario Test Execution Guide<!-- omit from toc -->

## Table of Contents<!-- omit from toc -->

- [Introduction](#introduction)
- [Scenario Execution](#scenario-execution)
  - [Using a Ref](#using-a-ref)
  - [Using a Chain](#using-a-chain)

## Introduction

> This document aims to define and guide you through creating the "Execution" step of a scenario.

Reviewer: ```suggestion
The purpose of this document is to define and guide you through creating the "Execution" step of a scenario.
```

> Please follow the guide below to create the execution step of your scenario.

Reviewer 2: Given that you capitalize "Execution" earlier in this document when discussing the Execution step, we should be consistent here.

Author: ```suggestion
Please follow the guide below to create the Execution step of your scenario.
```

Author: Resolved in [**39fe241**7dca0aa2df5e3e0cb4676f4fa7ac99bf7](https://github.com/CSPI-QE/ocp-ci-docs/pull/10/commits/39fe2417dca0aa2df5e3e0cb4676f4fa7ac99bf7)

> Please follow the guide below to create the execution step of your scenario.

Author: ```suggestion
Please follow the guide below to create the Execution step of your scenario.
```
